VetClinicApi.Application.Tests/Services/AmimalHandling/AnimalService_Test.cs

```diff
@@ -27,25 +27,25 @@ public void Create_ValidAnimal_Test()
             Breed = "Test2",
             IsVaccinated = true
         };
-        _animalRepository.Setup(x => x.Add(It.IsAny<Animal>())).Returns(animal);
+        _animalRepository.Setup(x => x.Add(It.IsAny<Animal>())).ReturnsAsync(animal);
 
         var animalService = new AnimalService(_animalRepository.Object, _animalTypeRepository.Object);
-        var result = animalService.CreateAnimal(animal);
+        var result = await animalService.CreateAnimal(animal);
 
         Assert.Equal(DateTime.Today, result.RegistrationDate);
         Assert.Equal(DateTime.Today, result.LastEditDate);
     }
 
     [Fact]
-    public void Create_AnimalIsNull_Test()
+    public async Task Create_AnimalIsNull_Test()
     {
         var animalService = new AnimalService(_animalRepository.Object, _animalTypeRepository.Object);
 
-        Assert.Throws<ArgumentNullException>(() => animalService.CreateAnimal(null));
+        Assert.ThrowsAsync<ArgumentNullException>(() => animalService.CreateAnimal(null));
```

Comment:
```suggestion
        await Assert.ThrowsAsync<ArgumentNullException>(async () => await animalService.CreateAnimal(null));
```

And so on for every ThrowsAsync